toolkit/tools/scheduler/scheduler.go

```diff
@@ -120,24 +121,25 @@ func main() {
 		logger.Log.Fatalf("Unable to generate package build list, error: %s", err)
 	}
 
-	var reservedFiles []string
-	if len(reservedFileListFile) > 0 {
-		reservedFiles, err = schedulerutils.ReadReservedFilesList(reservedFileListFile)
+	var toolchainPackages []string
+	if len(toolchainManifest) > 0 {
+		toolchainPackages, err = schedulerutils.ReadReservedPackageManifest(toolchainManifest)
```

Comment:
I would consider changing the name of this function as well. The only times you use it is to process toolchain manifest, so might as well name it `ReadToolchainPackageManifest` or similar.

Reply:
Fair, updated.